test:
  ros__parameters:
    environment: 'CarTrack1'
    max_steps_evaluation: 1000000
    actor_path: models/cartrack_1_training-26-06-2023-06:50:49_150000_actor.pht
    critic_path: models/cartrack_1_training-26-06-2023-06:50:49_150000_critic.pht